verilog/CST/functions.h

```diff
@@ -111,6 +111,19 @@ const verible::Symbol* GetFunctionReturnType(
 // Returns the id of the function declaration.
 const verible::Symbol* GetFunctionId(const verible::Symbol& function_decl);
 
+// Returns leaf node for function name.
+// e.g. function my_fun(); return leaf node for "my_fun".
+const verible::SyntaxTreeLeaf* GetFunctionName(const verible::Symbol&);
+
+// Returns leaf node for function name in function call.
+// e.g my_function(); return leaf node for "my_function".
+const verible::SyntaxTreeLeaf* GetFunctionCallName(
+    const verible::Symbol&);
+
+// Returns the function declaration body.
+const verible::SyntaxTreeNode& GetFunctionBlockStatementList(
```

Comment:
Sorry, just noticed.  Can you add unit-tests for this function?

Reply:
We'll merge this now, if you can start a PR to add the missing tests.

Reply:
@fangism  I added the tests. Is this what you asked for?